target/linux/ath79/generic/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -199,13 +200,8 @@ case "$FIRMWARE" in
 		ln -sf /lib/firmware/ath10k/pre-cal-pci-0000\:00\:00.0.bin \
 			/lib/firmware/ath10k/QCA9888/hw2.0/board.bin
 		;;
-	tplink,tl-wpa8630p-v2-eu|\
```

Comment:
These were added in master before the eap245-v3 got merged (9dd4ba3d7ed56413399b1e36f810813c1dcf7473) and I must have missed them on a rebase. The statements are just moved down to fix the sorting, but I can turn this into a separate patch if that's preferred.

Reply:
Note this patch which will sort them anyway: https://patchwork.ozlabs.org/project/openwrt/patch/20200912195248.47681-1-freifunk@adrianschmutzler.de/

So, maybe just ignore them for now.

Reply:
Yes, I saw your patch this morning on the mailing list. Good idea to harmonise that stuff, I noticed it too while cleaning up the patches for the PR. Please go ahead with your patch first, then I'll rebase on top of that.

Reply:
Rebased the patch on master, so now it only adds the line for the EAP225v3 again.